Automattic/wp-calypso
# Update site creation labels and an option

#### Changes proposed in this Pull Request

* Adds an option to the create site call for P2s.
* Changes the wording on the P2 create new site form.

#### Testing instructions

* Go to http://calypso.localhost:3000/start/p2 and verify that the labels on the form have changed.
* Create a new site by filling out the form and verify that the `p2_initialize_as_hub` option is passed when submitting.



*



Translators: The form with `Name your team` will look like this:
<img width="461" alt="Screen Shot 2021-06-03 at 14 57 27" src="https://user-images.githubusercontent.com/193283/120648739-45bdfd80-c47c-11eb-9b4d-5d0a13fce584.png">
